libs/langgraph/src/errors.ts

```diff
@@ -97,3 +97,20 @@ export class InvalidUpdateError extends Error {
     return "InvalidUpdateError";
   }
 }
+
+export class MultipleSubgraphsError extends Error {
+  constructor(message?: string) {
+    super(message);
+    this.name = "MultipleSubgraphError";
+  }
+
+  static get unminifiable_name() {
+    return "MultipleSubgraphError";
+  }
+}
+
+/**
+ * Used for subgraph detection.
+ * TODO: Move to globalThis for bundlers
```

Comment:
ooc: what's blocking us from doing now?

Reply:
Feels premature and don't want to rely on globals if possible, maybe would be a good idea.